package.json

```diff
@@ -44,7 +44,7 @@
     "clean": "rimraf 'node_modules/.cache' '.awcache' 'dist'",
     "prerelease": "npm run build",
     "release": "./config/deployment/release.sh",
-    "test": "jest --detectOpenHandles",
+    "test": "NODE_ICU_DATA=node_modules/full-icu jest --detectOpenHandles",
```

Comment:
It is a good solution, but there is one downside.

You need to pass an additional environment variable before launching jest.

So it should be configured manually in any tool, which uses jest.

e.g., if I use jest extension for vscode, I need to configure it there
if I use WebStorm, I need to configure it there.

Another solution, that I have found is to use `intl` fallback. Just file with `global.Intl = require("intl");` in jest's `setupFile` and this fallback will be used across all the tests providing Intl with all locales.

Reply:
https://www.npmjs.com/package/intl